INSTALL.md

```diff
@@ -38,11 +57,13 @@
        A. **Using ACL on a system that supports chmod +a**
 
        ```bash
-       $ rm -rf ezpublish/cache/* ezpublish/logs/* ezpublish/sessions/*
+       $ rm -rf ezpublish/cache/*
+       $ rm -rf ezpublish/logs/*
+       $ rm -rf ezpublish/sessions/*
```

Comment:
Should (rebase) make sure to keep this as one line like it is now in ezplatform.